docs/source/osds/scenarios.rst

```diff
@@ -187,10 +187,19 @@ is only available when the ceph release is Luminous or newer.
    The creation of the logical volumes is not supported by ``ceph-ansible``, ``ceph-volume``
    only creates OSDs from existing logical volumes.
 
-``lvm_volumes`` is the config option that needs to be defined to configure the
-mappings for devices to be deployed. It is a list of dictionaries which expects
-a volume name and a volume group for logical volumes, but can also accept
-a partition in the case of ``filestore`` for the ``journal``.
+``lvm_volumes`` or ``devices`` are the config option that needs to be defined to deploy OSDs
+with the ``lvm`` osd scenario.
+
+``lvm_volumes`` is a list of dictionaries which expects a volume name and a volume group for
+logical volumes, but can also accept a partition in the case of ``filestore`` for the ``journal``.
+If ``lvm_volumes`` is defined then the ``ceph-volume lvm create`` command is used to create each OSD
+defined in ``lvm_volumes``.
+
+``devices`` is a list of raw device names as strings. If ``devices`` is defined then the ``ceph-volume lvm batch``
+command will be used to deploy OSDs.
+
+Both ``lvm_volumes`` and ``devices`` can be defined and both methods would be used in the deployment or you
+can pick just one method.
```

Comment:
I am on the fence with this one, is it useful to allow both? From the ceph-ansible side of things, is it easier to just allow them vs. picking one or the other?

Reply:
Why not allow both? It is easier from the ceph-ansible side to allow them both. It means less conditions in the ``when`` clause anyway when including the scenario files.

Reply:
Yeah we can allow that, was just surprised to see both.